### PR TITLE
fix(nav & pagelayout): removes basic nav example, switches to default variant,  updates PF masthead logo

### LINKED ISSUE #3044: Update nav examples and demos
https://github.com/patternfly/patternfly-next/pull/2387 

* on Navigation component examples
  * remove "basic" example
  * update "grouped" exmaple to use default navigation instead of simple
* on Page Layout demos
  * update "grouped nav" example to use default nav instead of simple
  * remove "basic nav" example

Also need to update this patternfly logo (any demos, examples, and the default page at https://patternfly-react.surge.sh/) to use the new SVG logo you can find here - https://pf4.patternfly.org/assets/images/PF-Masthead-Logo.svg

Screen Shot 2019-09-16 at 2 32 31 PM

Screen Shot 2019-09-16 at 2 34 47 PM

### PR DESCRIPTION
**Pull request by jenny-s51**
Closes #3044

### CODE REVIEW

File: .circleci/index.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
           </button>
         </div>
         <a class="pf-c-page__header-brand-link" href="#">
-          <img class="pf-c-brand pf-m-sm" src="https://pf4.patternfly.org/assets/images/l_pf-reverse-164x11.png" alt="Patternfly Logo" />
+          <img class="pf-c-brand pf-m-sm" src="https://pf4.patternfly.org/assets/images/PF-Masthead-Logo.svg" alt="Patternfly Logo" />
```
Comment (dlabrecq) [inline .circleci/index.html]: Curious how this image is being used? I don't see it in the live page source

File: packages/patternfly-4/react-core/src/components/Nav/NavGroup.tsx [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
 import styles from '@patternfly/react-styles/css/components/Nav/nav';
 import { css } from '@patternfly/react-styles';
+import { NavVariants } from './NavVariants';
```
Comment (boaz0) [inline packages/patternfly-4/react-core/src/components/Nav/NavGroup.tsx]: You imported `NavVariants` but you're not using it.

File: .circleci/index.html (hunk 1 above)
Comment (jenny-s51) [reply .circleci/index.html]: @dlabrecq This is how the new logo is used when I open this file in my browser: 
Screen Shot 2019-11-08 at 12 32 28 PM

File: packages/patternfly-4/react-core/src/components/Nav/NavGroup.tsx (hunk 2 above)
Comment (jenny-s51) [reply packages/patternfly-4/react-core/src/components/Nav/NavGroup.tsx]: nice catch -- fixed. thank you!

**Comment by mcoker** [comment]
Hey @jenny-s51 looks great! I missed one spot in the original issue - on https://patternfly-react-pr-3262.surge.sh/patternfly-4/documentation/react/demos/pagelayout, we need to remove the first (basic nav) example

File: packages/patternfly-4/react-core/src/components/Nav/NavGroup.tsx [hunk 3]
```diff
@@ -25,6 +25,6 @@ export const NavGroup: React.FunctionComponent<NavGroupProps> = ({
     <h2 className={css(styles.navSectionTitle)} id={id}>
       {title}
     </h2>
-    <ul className={css(styles.navSimpleList)}>{children}</ul>
+    <ul className={css(styles.navList)}>{children}</ul>
```
Comment (redallen) [inline packages/patternfly-4/react-core/src/components/Nav/NavGroup.tsx]: This matches what's in Core on https://pf4.patternfly.org/documentation/core/components/nav .

**Comment by mcoker** [comment]
@jenny-s51 sorry I missed one spot - can you also update the [legacy/light nav page demo](https://patternfly-react-pr-3262.surge.sh/patternfly-4/documentation/react/demos/pagelayout/legacylight-nav) to use the default nav instead of the simple nav?

**Pull request merged**